test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+#if NET461
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Messaging;
+#else
+using System.Threading;
+#endif
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public abstract class MSBuildIntegrationTestBase
+    {
+#if NET461
+#elif NETCOREAPP2_0 || NETCOREAPP2_1
+        private static readonly AsyncLocal<ProjectDirectory> _project = new AsyncLocal<ProjectDirectory>();
```

Comment:
Isn't this in netstandard2.0? Should work with net461, shouldn't it?

Reply:
no it is not in net461